### ISSUE TITLE
Adjust dependency versions

### ISSUE THREAD

**Issue by mdragaschnig** [blockchain-etl/bitcoin-etl#54]
Hi,

I noted that the current, fixed dependency versions of requests, dateutil and click clash with dependencies for e.g. pandas.

Would it be possible to adjust the Python dependency definitions in setup.py?

Locally I've successfully tested  

'requests ~= 2.20',
'python-dateutil ~= 2.7',
'click ~= 7.0'

**Comment by defconst** [comment]
@medvedev1088 any updates on this please?

**Comment by medvedev1088** [comment]
Yes it would be possible. If you create a PR I'll review and merge it
